src/editor/Store/CommandStoreConnector.re

```diff
@@ -87,6 +87,64 @@ let start = _ => {
         ]),
     ),
     ("quickOpen.open", _ => singleActionEffect(QuickOpen)),
+    (
+      "quickOpen.openFiles",
+      state => {
+        let currentDirectory = Rench.Environment.getWorkingDirectory();
+
+        let getDisplayPath = (fullPath, dir) => {
+          let re = Str.regexp_string(dir ++ Filename.dir_sep);
+          Str.replace_first(re, "", fullPath);
+        };
+
+        multipleActionEffect([
+          MenuOpen(
+            (setItems, _, _) => {
+              let commands =
+                state.Oni_Model.State.buffers
+                |> IntMap.to_seq
+                |> List.of_seq
```

Comment:
I would guess that using [Seq](https://caml.inria.fr/pub/docs/manual-ocaml/libref/Seq.html) directly would be a tiny bit more efficient. Probably doesn't matter much, but seems a bit cleaner to me as well.
Also, because of the duplication on `getMetadata` it might be usefull to use `Seq.filter_map`.

Reply:
Thanks @tcoopman. Ill make those changes.